infra/gcp/lib_gcr.sh

```diff
@@ -149,3 +149,22 @@ function empower_svcacct_to_admin_gcr () {
 
     empower_svcacct_to_admin_gcs_bucket "${group}" "${bucket}"
 }
+
```

Comment:
First note - this does not belong in this file.  This file just for very low-level GCR-centric primitives.  Put this back in lib.sh until that can be broken up more.